### PR TITLE
TW-1231 using x-api-key to generate QR code

### PR DESCRIPTION
**Pull request by vutrantokoin**
Ticket: https://tokoin.atlassian.net/browse/TW-1231

Expected:
- generate QR code

For production env, please use the API key `public_api_key-4bd08c47-0b74-4a3a-81e0-db358b537aa7`

### CODE REVIEW

**Review by BinhNguyenTokoin — CHANGES_REQUESTED** [review]
The sdk generates successfully but the my t wallet app crashed when scanning. I used dev branch for my t wallet 
tchainQR

https://user-images.githubusercontent.com/57206348/209757758-d3429b88-f3b6-4a68-b833-2c8ca1eb8568.mov

**Comment by vutrantokoin** [comment]
The crash's fixed in https://github.com/tokoinofficial/tokoin_wallet/pull/417. Please check this PR again

File: example/lib/utils/constants.dart [hunk 1]
```diff
@@ -1,4 +1,4 @@
 class Constants {
-  static const String apiKey = '3e093592-3e0e-4a52-9601-ead49f794586';
+  static const String devApiKey = '3e093592-3e0e-4a52-9601-ead49f794586';
```
Comment (BinhNguyenTokoin) [inline example/lib/utils/constants.dart:2]: I think apiKey is a better name, user will receive prod api key from email and paste here. Using the devApiKey looks confused

**Review by BinhNguyenTokoin — APPROVED** [review]
Tested on dev full flow, tested on prod - stop at success to scan only 
Only one comment, fix that then we are good to carry on

**Pull request merged**